### PR TITLE
Add feature skip_roothints_download

### LINKED ISSUE #103: Option not to use/download root.hints
Having a purely caching forwarder which just forwards request to some other recursive nameserver there is no need for the root.hints file and in our case the servers can't even reach the Internet so it hangs on downloading the root.hints (until touching it manually). Please create an option not to use/download root.hints.

### PR DESCRIPTION
**Pull request by mrdima**
An option to skip the download of root.hins. There are many ways, I wanted to change as little as possible. Closes #103

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -100,6 +102,12 @@
     include unbound::remote
   }
 
+  if $skip_roothints_download {
+    File[$hints_file] -> Exec['download-roothints']
+  } else {
+    Exec['download-roothints'] -> File[$hints_file]
+  }
```
Comment (igalic) [inline manifests/init.pp]: i like this hack ;)

**Comment by igalic** [comment]
i was gonna say: can you add a spec tests, but we don't have any spec tests for this class now (other than  whether it exists), so it wouldn't be fair on you…

you should however document the feature in the readme!

**Comment by mrdima** [comment]
Yes, I thought about the readme after submitting, I'll submit that. Yeah, rspec and even general parameter validation is lacking in this module

**Comment by mrdima** [comment]
I tried to keep the documentation in the style of the readme, it's a bit different from regular modules which usually document all the parameters, hope this is ok.

**Comment by igalic** [comment]
super! thank you @mrdima!

**Pull request merged**